devito/types/equation.py

```diff
@@ -158,9 +158,9 @@ def __str__(self):
     __repr__ = __str__
 
 
-class LessThan(sympy.LessThan):
+class Ne(sympy.Ne):
```

Comment:
Note: To discuss how to relate or not with CondNe.

Reply:
we don't